README.md

```diff
@@ -223,134 +226,134 @@ have some inaccuracies in number parsing:
-|             C#/.NET Core |    8.697<sub>±0.219</sub> |     34.63<sub>±00.12</sub> + 69.39<sub>±00.05</sub> |    222.21<sub>±22.74</sub> |
-|                  C#/Mono |   13.592<sub>±0.646</sub> |     20.42<sub>±00.04</sub> + 69.01<sub>±00.02</sub> |    367.41<sub>±31.99</sub> |
-|         Ruby/truffleruby |   30.741<sub>±2.113</sub> | 329.32<sub>±17.21</sub> + 1016.19<sub>±371.12</sub> |   717.20<sub>±127.90</sub> |
-| Ruby/truffleruby (--jvm) |   51.371<sub>±1.132</sub> |  400.60<sub>±12.24</sub> + 638.52<sub>±151.81</sub> |  1294.26<sub>±132.95</sub> |
-|             Ruby (--jit) |  246.812<sub>±6.744</sub> |    272.22<sub>±00.01</sub> + 68.84<sub>±00.00</sub> |  6126.62<sub>±145.10</sub> |
-|                     Ruby |  258.210<sub>±7.250</sub> |     15.64<sub>±00.04</sub> + 68.58<sub>±00.00</sub> |  6630.94<sub>±231.12</sub> |
-|                   Python |  277.581<sub>±2.827</sub> |     10.33<sub>±00.03</sub> + 68.58<sub>±00.00</sub> |   6803.39<sub>±91.67</sub> |
-|                      Tcl |  385.609<sub>±3.409</sub> |     7.37<sub>±00.02</sub> + 400.39<sub>±00.02</sub> |   9718.19<sub>±92.19</sub> |
-|                     Perl |  472.250<sub>±5.819</sub> |     9.57<sub>±00.03</sub> + 599.61<sub>±00.05</sub> |  12162.42<sub>±90.60</sub> |
-|               Ruby/jruby | 686.602<sub>±38.075</sub> |   264.80<sub>±08.62</sub> + 672.79<sub>±76.03</sub> | 16661.14<sub>±977.64</sub> |
+|                 Language |                   Time, s |                                       Memory, MiB |                  Energy, J |
+| :----------------------- | ------------------------: | ------------------------------------------------: | -------------------------: |
+|          D/ldc2 (lubeck) |    0.046<sub>±0.000</sub> |    6.57<sub>±00.04</sub> + 54.95<sub>±00.10</sub> |      4.66<sub>±00.03</sub> |
+|    Nim/gcc (Arraymancer) |    0.066<sub>±0.002</sub> |    5.52<sub>±00.05</sub> + 54.73<sub>±00.11</sub> |      5.46<sub>±00.13</sub> |
+|      C++/clang++ (Eigen) |    0.068<sub>±0.003</sub> |   38.90<sub>±02.59</sub> + 51.51<sub>±02.58</sub> |      5.53<sub>±00.15</sub> |
+|           Python (NumPy) |    0.070<sub>±0.000</sub> |   28.41<sub>±00.12</sub> + 54.96<sub>±00.04</sub> |      6.49<sub>±00.04</sub> |
+|          C++/g++ (Eigen) |    0.070<sub>±0.002</sub> |   29.55<sub>±08.72</sub> + 60.24<sub>±08.76</sub> |      5.13<sub>±00.13</sub> |
+|  Nim/clang (Arraymancer) |    0.072<sub>±0.001</sub> |    6.34<sub>±00.04</sub> + 54.76<sub>±00.08</sub> |      6.07<sub>±00.07</sub> |
+|              Java (ND4J) |    0.081<sub>±0.005</sub> |  104.24<sub>±01.83</sub> + 92.25<sub>±00.16</sub> |      6.35<sub>±00.31</sub> |
+|           Rust (ndarray) |    0.090<sub>±0.000</sub> |    2.63<sub>±00.04</sub> + 68.48<sub>±00.00</sub> |      6.00<sub>±00.03</sub> |
+|       Julia (threads: 4) |    0.191<sub>±0.000</sub> |  235.35<sub>±00.21</sub> + 49.23<sub>±00.90</sub> |     15.54<sub>±00.02</sub> |
```

Comment:
wonder why julia slow?

Reply:
The server has Intel Xeon processor with 4 cores only (https://www.intel.ca/content/www/ca/en/products/sku/212255/intel-xeon-e2324g-processor-8m-cache-3-10-ghz/specifications.html), the previous testing infrastructure used desktop processor with the higher number of cores. OpenBLAS automatically setup the number of threads to use based on the number of CPU cores, so the previous results were using 8 threads, but this one - 4 threads.